athena-elasticsearch/src/main/java/com/amazonaws/connectors/athena/elasticsearch/ElasticsearchRecordHandler.java

```diff
@@ -64,6 +66,10 @@
     // is external to Amazon (false), and the domain_mapping environment variable should be used instead.
     private static final String AUTO_DISCOVER_ENDPOINT = "auto_discover_endpoint";
 
+    // Env. variable that holds the query timeout period for the Search queries.
+    private static final String QUERY_TIMEOUT_SEARCH = "query_timeout_search";
+    private Long queryTimeout;
```

Comment:
final and should be long not Long. This site explains a bit about java 'Boxing' (e.g. Long vs long), incase that concept is new to you since Java is relatively new to you. https://www.baeldung.com/java-primitives-vs-objects

Reply:
Changed to final long.